chains/irisnet/handlerIrisnet.go

```diff
@@ -138,6 +155,12 @@ func (h *TendermintHandler) upgradeConnectionAndHandshake() error {
 	return nil
 }
 
+// handshake function follows the protocol set on amino spec,
+// MarshalBinaryLengthPrefixed encodes the object according to the Amino spec
```

Comment:
don't mention exact amino codec setup. We just need to know that how handshake happens